src/ocamlorg_frontend/components/learn_overview.eml

```diff
@@ -0,0 +1,13 @@
+let lang_manual_banner =
+  <div class="container bg-learn-area-orange w-full h-80 flex items-center px-4">
+    <div class="w-2/3 pr-4 mb-2">
```

Comment:
container needs to be inside the surrounding content, and don't use `container`, but `container-fluid` is probably more appropriate here.